ui/pages/institutional/custody/custody.js

```diff
@@ -74,9 +76,10 @@ const CustodyPage = () => {
   const [apiUrl, setApiUrl] = useState('');
   const [addNewTokenClicked, setAddNewTokenClicked] = useState(false);
   const [chainId, setChainId] = useState(parseInt(currentChainId, 16));
-  const [connectRequest, setConnectRequest] = useState(undefined);
+  const connectRequests = useSelector(getInstitutionalConnectRequests, isEqual);
```

Comment:
Using connectRequests from the store instead of getting them from `mmiActions.getCustodianConnectRequest()` as we were having some state mutations